products/kindle.md

```diff
@@ -62,6 +62,8 @@ releases:
     latest: "5.12.2.2"
     releaseDate: 2013-09-30
 permalink: /kindle
+alternate_urls:
+-   /amazon-kindle
```

Comment:
now about we make products in same stucture ? i mean lets use `company-product` structure for its main permalink but use `product` as in alternate_urls
[ i mean lets change amazon-kindle as permalink but kindle as alternate_urls  (just like in eks) ]

Reply:
@marcwrobel

Reply:
We don't advertise shortlinks prominently, so if it's possible (not confusing) for us to use shortlinks that are memorable as primary URLs, we should.

Reply:
Ok, so let's keep the shorter URL as primary link and merge this PR.